pino-eventhub.js

```diff
@@ -102,6 +105,15 @@ function pinoEventHub (opts) {
         req.on('error', (e) => {
           console.error(`request error =`, e)
         })
+        req.on("socket", function (socket) {
+          if (sockets.indexOf(socket) === -1) {
+            debug("new socket created");
+            sockets.push(socket);
+            socket.on("close", function() {
+                debug("socket has been closed");
+            });
+          }
+        });
```

Comment:
I was just using it to show that the socket was being reused.  I can remove that.